packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -188,6 +189,59 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	}
 }
 
+async function downloadWordPressAssets(php: PHP) {
+	if (!php.requestHandler) {
+		logger.warn('No PHP request handler available');
+		return;
+	}
+	const wpVersion = await getLoadedWordPressVersion(php.requestHandler);
+	const staticAssetsDirectory = wpVersionToStaticAssetsDirectory(wpVersion);
+	if (!staticAssetsDirectory) {
+		logger.warn('No static assets directory for WordPress', wpVersion);
+		return;
+	}
+	const response = await fetch(
+		[wordPressSiteUrl, staticAssetsDirectory, 'wordpress-static.zip'].join(
+			'/'
+		)
+	);
+
+	if (!response.ok) {
+		logger.warn('Failed to download WordPress assets');
```

Comment:
This kind of if(error) { log(); return; } is useful in lower level languages where we risk accessing a null pointer crashing the entire program. In JavaScript we can be way less defensive and just allow the browser to throw when accessing a null response body object. That would also make the error clear for the caller - right now the result of calling this function is the same whether is succeeds or fails.